indico/modules/events/abstracts/templates/emails/comment.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'events/abstracts/emails/base.html' %}
+
+{% block subject_message %}New abstract comment{% endblock %}
+
+{% block header_subtitle %}New abstract comment{% endblock %}
+
+{% block header_subtitle_ref %}ref #{{ abstract.friendly_id }}{% endblock %}
+
+{% block content_header %}
+    <p>Dear {{ salutation }},</p>
```

Comment:
I would avoid this; it'll be a huge pain once we add i18n to emails... How about just using `Dear <firstname>`?

Reply:
I wanted to avoid sending individual emails, but with the i18n we'll probably have to do it anyway.. :+1: